circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   xcode:
-    version: 8.2
+    version: 8.3
```

Comment:
Ah, cool, didn't check the CI output.

It's weird though that our script isn't finding Xcode.